tests/test_suite.html

```diff
@@ -743,6 +743,34 @@
             w14Running.should.equal(false)&&w15Running.should.equal(true);
         });
 
+        // Recipe-level sampleRate property must be defined for all recipes if it is being used
+        var w16Activated = false;
+        try {
+            Mojito.addTest({
+                id: 'w16',
+                state: 'staging',
+                name: 'recipe_level_sampleRate',
+                sampleRate: 1,
+                trigger: function (e) {
+                    w16Activated = true;
+                    e.activate();
+                },
+                recipes: {
+                    '0': {
+                        name: 'control',
+                        sampleRate: 0.5
+                    },
+                    '1': {
+                        name: 'treatment'
+                    }
+                }
+            });
+        } catch(e) {
+            it('Recipe-level sampleRate property must be defined for all recipes if it is being used', function () {
+                w16Activated.should.equal(false);
+            });
+        }
```

Comment:
If no error is thrown, I think this unit test is never run. e.g. if the w16Activated variable is set to true and no error is thrown, the catch statement is not evaluated:

<img width="795" alt="Screen Shot 2021-03-18 at 18 22 29" src="https://user-images.githubusercontent.com/2361388/111588034-20273d80-8817-11eb-9067-8c678bc02a19.png">

Should it sit outside the try..catch statement?

Reply:
Oh and while we're there, let's give the test a proper name like:

`Tests fail when recipe-level sample rates are not set for all recipes.`

Reply:
Thanks @kingo55 , you're right, it should sit outside the  try..catch block. 
Fixed and updated the test name.

Reply:
Awesome... Thanks @allmywant - looking good now!

Want to do the honours and merge/squash?

Reply:
@kingo55 
Merged. It seemed I should do `git merge squash` first, I remember you shared an article about it but I forgot the URL, can you please share it again?